### ISSUE TITLE
Playground Laboratories

### ISSUE THREAD

**Issue by b300098957** [NativeScript/playground-feedback#182]
Hi,

I am a college professor using Nativescript Playground to introduce one of my mobile courses.

Teenagers like the Playground running on their phones and are very much engaged.

The only thing is I can interact with the playground to evaluate their work (all of them are now remotely learning from home)

Are there any lab services I can hook to to automatically evaluate/grade the learners without having to ask them to download their code and email or dump it to a remote VCS?

Thanks,

Remote lab examples integrated to MOOCs (i.e. Coursera, ...)
https://www.qwiklabs.com/
https://aws.amazon.com/cloud9/
https://azure.microsoft.com/en-us/services/lab-services/

**Comment by EdJones** [comment]
I'm not clear what you're asking here. They're publishing their app in Playground, and you can see the code, and also see how it runs? But you'd like someone else to review the code?

**Comment by setrar** [comment]
> But you'd like someone else to review the code?

Correct, I would like someone but ideally a robot (i.e. github actions) to grade the code.
I would then base all my Nativescript labs on the playground laboratory instead of using github actions as the lab grader.

https://github.com/CollegeBoreal/INF1083-200-19A-01/tree/master/2.Templates/b300098957-blank-ng/src/tests

**Comment by b300098957** [comment]
> But you'd like someone else to review the code?

Correct, I would like someone but ideally a robot (i.e. github actions) to grade the code.
I would then base all my Nativescript labs on the playground laboratory instead of using github actions as the lab grader.

https://github.com/CollegeBoreal/INF1083-200-19A-01/tree/master/2.Templates/b300098957-blank-ng/src/tests
